configure-box.sh

```diff
@@ -11,6 +11,17 @@
 # Install the docker-compose box to the current working directory
 # Pre-requisites: wget
 
+check_dependencies () {
+  for cmd in "$@"; do
+    if ! command -v $cmd >/dev/null 2>&1; then
+      echo "This script requires \"${cmd}\" to be installed"
+      exit 1
+    fi
+  done
+}
+
+check_dependencies wget docker
```

Comment:
Can you also check for `docker-compose` here?

Reply:
Actually docker-compose isn't required to run the install. Everything can be set up without it.

The main purpose of this is to make it easy to install and configure.